backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/model/validator/M_InOut.java

```diff
@@ -134,7 +133,8 @@ public void destroyHandlingUnitsForReceipt(final I_M_InOut inout)
 	@DocValidate(timings = { ModelValidator.TIMING_BEFORE_PREPARE })
 	public void addPackingMaterialLinesForShipment(final I_M_InOut shipment)
 	{
-		if (shipment.isSOTrx())
+		final MovementType movementType = MovementType.ofCode(shipment.getMovementType());
+		if (movementType.isOutboundTransaction() && shipment.isSOTrx())
```

Comment:
do we need to check the movementtype and IsSOTrx.
Just checking the MovementType (or EmptiesMovementDirection) wouldn't be enough?